docs/modules/ROOT/pages/how-tos/worklows/git-submodules.adoc

```diff
@@ -0,0 +1,101 @@
+= Git Submodule guideline
+
+This document outlines some basic guidelines for synchronizing changes between an upstream and downstream repository using Git submodules.
+
+By leveraging Git submodules, it is possible to maintain a consistent and up-to-date relationship between related repositories (upstream and downstream) while having more control of how the project is built, tested and released.
+
+Additionally, the integration with Konflux can automate key aspects of this process, ensuring that changes from the upstream repository are efficiently and reliably propagated to the downstream repository.
```

Comment:
What is "the integration"?

Reply:
I removed "the integration" and left it as "Konflux" so it's less confusing

Reply:
We use `{ProductName}` instead of Konflux and then the docs will render it.

Reply:
Good catch! Let me edit it...

Reply:
I might be mistaken, but it looks like I did use `{ProductName}` -> https://github.com/konflux-ci/docs/pull/120/files#diff-8168286ae4fbd44cc1f9e5677aa3a5949ae97f5f391b1466cc253efe6176ccf8R7

Reply:
I must have gotten confused with an older revision, apologies.